index.html

```diff
@@ -1,19 +1,124 @@
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <title>HTML-CSS-Module-Project</title>
+  </head>
+  <body>
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <header class="header">
+      <img
+        src="img/karma-logo.svg"
+        alt="it is used for website"
+        class="header__logo"
+      />
+      <nav>
+        <ul class="nav-list">
+          <li class="nav__item">
+            <a class="nav__link" href="#">Meet Karma</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">How it Works</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Store</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Blog</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Help</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Login</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <content class="content">
+      <content class="introduction">
+        <h1>Introduction Karma</h1>
+        <p>Bring WiFi with you, everywhere you go</p>
+        <main class="btn">
+          <button>Learn More</button>
+        </main>
+      </content>
+
+      <content class="main">
+        <p>Everyone Needs a little Karma</p>
+      </content>
+
+      <section class="more__caption">
+        <article class="art-area">
+          <img
+            class="article__img"
+            src="img/icon-devices.svg"
+            alt="more detail"
+          />
+          <p class="art-text">Internat for all devices</p>
+        </article>
+
+        <article class="art-area">
+          <img
+            class="article__img"
+            src="img/icon-coffee.svg"
+            alt="you can enjoy your spare time."
+          />
+          <p class="art-text">Boost your productivity</p>
+        </article>
+
+        <article class="art-area">
+          <img
+            class="article__img"
+            src="img/icon-refill.svg"
+            alt="you can visit anytime "
+          />
+          <p class="art-text">Pay us You Go</p>
+        </article>
+      </section>
+    </content>
+
     <!-- All the images you need are in the 'img' folder -->
 
-</body>
+    <hr class="new" />
+
+    <footer class="footer">
+      <p>Join us own</p>
+      <content class="footer__content">
+        <img
+          class="img__footer"
+          src="img/twitter-icon.svg"
+          alt="you can have contact"
+        />
+
+        <img
+          class="img__footer"
+          src="img/facebook-icon.svg"
+          alt="for more detail"
+        />
+
+        <img
+          class="img__footer"
+          src="img/instagram-icon.svg"
+          alt="you can visit"
+        />
```

Comment:
Just like your navigation bar, it would be beneficial to consider adding links to the social icons.